public/app/core/components/AppChrome/TopSearchBar.tsx

```diff
@@ -18,6 +20,7 @@ import { TOP_BAR_LEVEL_HEIGHT } from './types';
 export function TopSearchBar() {
   const styles = useStyles2(getStyles);
   const navIndex = useSelector((state) => state.navIndex);
+  const notifications = useSelector((state) => selectWarningsAndErrors(state.appNotifications));
```

Comment:
I think it's better if we use `selectAll` for the hackathon, because we might send `info` notifications from plugins, which then wouldn't show up here.